draft-ietf-git-using-github.md

```diff
@@ -419,6 +435,239 @@ A working group that uses GitHub MAY provide either facility at the request of
 the chairs.
 
 
+# Typical Working Group Policies
+
+Current experience with use of GitHub suggests two different approaches to
+greater use of the tool in Working Groups.
+
+This section describes the general approach taken in each case and discusses the
+consequences of a decision to adopt that particular model.  Working Groups can
+adjust these policies to suit their needs, but are advised to avoid gratuitous
+changes for the sake of consistency across the IETF as a whole.
+
+These approaches differ primarily in how discussion of substantive matters is
+managed.  Most of the advice in this document applies equally to both models.
+
+
+## Light GitHub Usage
+
+In a "light" mode of interaction, GitHub repositories are used to manage changes
+to documents, but the bulk of the work is conducted using email, face-to-face
+meetings, and other more traditional interactions.  The intent of this policy is
+to enable document and issue management using GitHub while minimizing the
+complexity of the process.
+
+The working group uses GitHub for tracking outstanding issues, but discussion of
+any substantial matters is always conducted on mailing lists.  Under this mode,
+issues and pull requests can be opened by anyone, but anything deemed
+substantive MUST resolved exclusively on the mailing list.  Discussion on GitHub
+is kept to a minimum.  Only editorial matters can be resolved using the issue
+tracker.
+
+Chairs and editors are given discretion in determining what issues are
+substantive.  As documents mature, it is generally prudent to err more toward
+consulting the mailing list where there is doubt.  As with other Working Group
+decisions, chairs are the arbiters in case of dispute.
+
+A recurrent problem with this mode of interaction is the tendency for
+discussions to spontaneously develop in the issue tracker.  This requires a
+degree of discipline from chairs and editors to ensure that any substantive
+matters are taken to the mailing list.
+
+As the Working Group mailing list remains the primary venue for discussion, this
+mode remains most compatible with existing work modes.  Participants in a
+Working Group that operates under this model can reasonably be expected to
+receive all relevant communication about the work of the group from the mailing
+list.
+
+Though the mailing list is used for making decisions, the issue tracker can
+still be a useful record of the state of issues.  It is often useful if chairs
+or editors record details of decisions in issue comments when closing issues as
+resolved.
+
+
+## Heavy GitHub Usage
+
+The "heavy" interaction mode differs from the "light" mode in that discussion
+relating to substantive technical matters is allowed to occur on GitHub issues.
+As a result, participants are permitted to contribute to discussion on the issue
+tracker.  In some cases, this can include making some decisions without
```

Comment:
This is going to be cause a lot of ire. I'd suggest at least parenthetically saying that those decisions are confirmed on the list. I know it's said below, but people will read this out of context (or just complain before reading onwards).